.stylelintrc

```diff
@@ -4,6 +4,8 @@
 		"stylelint-order"
 	],
 	rules: {
+		"block-opening-brace-newline-after": "always",
+        "block-closing-brace-newline-before": "always",
```

Comment:
Seems like this file needs to be added to the JSON linting.

Reply:
Resolved in 7319e5b.